evm2wast.h

```diff
@@ -0,0 +1,41 @@
+#ifndef __EVM2WAST_H
+#define __EVM2WAST_H
+#include <stdlib.h>
+#include "gadgets.h"
```

Comment:
Does gadgets needs to be included in the external interface?

Reply:
nope

Reply:
Can you remove it in that case?